examples/node_pool/variables.tf

```diff
@@ -54,20 +54,22 @@ variable "compute_engine_service_account" {
 
 variable "cluster_autoscaling" {
   type = object({
-    enabled             = bool
-    autoscaling_profile = string
-    min_cpu_cores       = number
-    max_cpu_cores       = number
-    min_memory_gb       = number
-    max_memory_gb       = number
+    enabled = bool
+    //Enable when autoscaling_profile becomes GA
+    //autoscaling_profile = string
```

Comment:
@bharathkkb I commented out `autoscaling_profile` because it is a beta feature. Comment is there so that we know to add this back once it becomes GA. If you think that this is not necessary then I will remove it.

Reply:
I think in that case its better to switch back to `"../../modules/beta-public-cluster/"` and keep the test as is. We have a couple of tests for the others in GA and we dont need to be exhaustive.